### ISSUE TITLE
Suggestion - make pots recolorable

### ISSUE THREAD

**Issue by Wakatsu** [Darkhax-Minecraft/BotanyPots#110]
Title kinda says it. It's a shame you have to craft a pot anew if you just want a different color pot.

**Comment by Darkhax** [comment]
This is a great idea, thanks for the feedback.

**Comment by sarifynna** [comment]
yes, it would be nice to just recolor pots by clicking with a dye or something?

**Comment by jengendler** [comment]
Agreed! I would LOVE the ability to click a pot with dye to re-color it. That way I won't have to break and replace pots that already have things growing in them. I would find that more useful than re-coloring in a crafting table.

**Comment by RobertLSnead** [comment]
This would be great!  It doesn't help when I thought you could do this prior and make a stack of them, doh, and not be able to recolor them.

**Comment by moshner** [comment]
+1
